dracut.sh

```diff
@@ -1551,6 +1562,10 @@ if [[ $hostonly ]] && [[ $hostonly_default_device != "no" ]]; then
                 for i in $(btrfs_devs "$_m"); do
                     push_host_devs "$i"
                 done
+            elif [[ $_t == zfs ]]; then
+                for i in $(zfs_devs "$_d"); do
+                    push_host_devs "$i"
+                done
```

Comment:
same applies here, afaict this can just be `push_host_devs $(zfs_devs "$_d")`; either way, nice, short, sweet, and infinitely better than the 02 we ship in openzfs now

Reply:
Made the change, and made a similar change for the other time `zfs_devs` is called.

Reply:
`shfmt` gave errors with this change, so I reverted it back to the way the `btrfs` stuff is implemented.